src/zcbenchmarks.cpp

```diff
@@ -116,7 +116,8 @@ double benchmark_create_joinsplit()
 
     struct timeval tv_start;
     timer_start(tv_start);
-    JSDescription jsdesc(*pzcashParams,
+    JSDescription jsdesc(false, // TODO: ?
```

Comment:
Add a commit that changes this to `true`, so that the benchmarks will run the new code.